[2022-03-09 18:13:11,150] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: tutorial.eaeaz.eaeaz_get_data scheduled__2022-03-07T10:29:52.784989+00:00 [queued]>
[2022-03-09 18:13:11,174] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: tutorial.eaeaz.eaeaz_get_data scheduled__2022-03-07T10:29:52.784989+00:00 [queued]>
[2022-03-09 18:13:11,177] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-03-09 18:13:11,180] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-03-09 18:13:11,200] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-03-09 18:13:11,224] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): eaeaz.eaeaz_get_data> on 2022-03-07 10:29:52.784989+00:00
[2022-03-09 18:13:11,233] {standard_task_runner.py:52} INFO - Started process 86 to run task
[2022-03-09 18:13:11,238] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 'eaeaz.eaeaz_get_data', 'scheduled__2022-03-07T10:29:52.784989+00:00', '--job-id', '669', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/tmp/tmp16en0lrf', '--error-file', '/tmp/tmpvvpat1xd']
[2022-03-09 18:13:11,240] {standard_task_runner.py:77} INFO - Job 669: Subtask eaeaz.eaeaz_get_data
[2022-03-09 18:13:11,367] {logging_mixin.py:109} INFO - Running <TaskInstance: tutorial.eaeaz.eaeaz_get_data scheduled__2022-03-07T10:29:52.784989+00:00 [running]> on host ff48034dfdbf
[2022-03-09 18:13:11,459] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=eaeaz.eaeaz_get_data
AIRFLOW_CTX_EXECUTION_DATE=2022-03-07T10:29:52.784989+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-07T10:29:52.784989+00:00
[2022-03-09 18:14:10,658] {taskinstance.py:1408} ERROR - Received SIGTERM. Terminating subprocesses.
